### PR TITLE
feat(ci): lighthouse

### PR DESCRIPTION
**Pull request by gunar**
- Adds a GitHub Action that runs Lighthouse on two flows (production and canary)
- Takes the opportunity to add a markdown heading to the Pizza sticky comment

### CODE REVIEW

**Comment by DafyddLlyr** [comment]
As discussed yesterday, I think it would be valuable to run this against a very simple canary flow (maybe under the testing team?) which has just a few nodes.

**Comment by gunar** [comment]
[Write automated tests that measure app performance](https://trello.com/c/OM1All6C/2132-write-automated-tests-that-measure-app-performance)

**Comment by DafyddLlyr** [comment]
Thanks for the canary! 🐤 Think you'll need to destroy Vultr and re-create if for the prod data to be pulled back in. Happy to approve and merge once that's done and I'm able able to just manually test the dummy flow.

**Comment by gunar** [comment]
@DafyddLlyr Done! Once this is merged, I believe the next step should be to measure the diff between PRs.

**Comment by gunar** [comment]
@theopensystemslab/planx I just remembered Daf is OOO—so this is open for anyone to review :)

**Review by DafyddLlyr — APPROVED** [review]
Great stuff - thanks for this. 

Agree that the sensible next step is measuring diffs on PR, so we can hopefully understand the impact of our changes.

**Pull request merged**
